keras_cv/utils/preprocessing.py

```diff
@@ -126,6 +126,11 @@ def blend(image1: tf.Tensor, image2: tf.Tensor, factor: float) -> tf.Tensor:
 
 
 def parse_factor(param, min_value=0.0, max_value=1.0, param_name="factor", seed=None):
+    if isinstance(param, dict):
+        # For all classes missing a `from_config` implementation.
```

Comment:
Would it make more sense to just implement `from_config` for these layers?

Reply:
This is a more temporary overarching fix for these layers. With the new serialization format, the serialization of keras.__internal__ namespace is not supported yet, but that will be added in the near future. When that happens, this line can be removed as well as many of the `from config` methods in other layers.

Reply:
sg -- can you please add a comment here to mention that this is temporary, explaining when it'll be safe to remove?

(Also the same in the PR description)